modules/rosa-sts-creating-a-cluster-quickly.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * rosa_getting_started_sts/rosa_creating_a_cluster_with_sts/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_quick_start/rosa-quick-start.adoc
 
 [id="rosa-sts-creating-cluster-using-defaults{context}"]
 = Creating a ROSA cluster with STS using the default options
```

Comment:
in here can we remove the two notes as they are not applicable to the quick start and the default options.  Having them here may cause confusion.

Reply:
Thanks for this suggestion. Resolved.